tests/Kerbero.Integration.Tests/NukiSmartLocksListIntegrationTests.cs

```diff
@@ -0,0 +1,156 @@
+using System.Net;
+using System.Net.Http.Json;
+using System.Text.Json.Nodes;
+using FluentAssertions;
+using Flurl.Http.Testing;
+using Kerbero.Infrastructure.Common.Context;
+using Kerbero.WebApi;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+
+namespace Kerbero.Integration.Tests;
+
+public class NukiSmartLocksListIntegrationTests: IDisposable
+{
+	private static readonly IConfigurationRoot Config = new ConfigurationBuilder()
+		.AddJsonFile("appsettings.Test.json")
+		.AddEnvironmentVariables()
+		.Build();
+
+	private readonly HttpTest _httpTest;
+	private readonly HttpClient _client;
+
+	public NukiSmartLocksListIntegrationTests()
+	{
+		WebApplicationFactory<Program> application = new KerberoWebApplicationFactory<Program>();
+		application.Server.PreserveExecutionContext = true; // fixture for Flurl
+		_httpTest = new HttpTest();
+		_client = application.CreateClient();
+	}
+	
+	public void Dispose()
+	{
+		_httpTest.Dispose();
+		GC.SuppressFinalize(this);
```

Comment:
```suggestion
		GC.SuppressFinalize(this);
```
perchè necessario chiamare il garbage collector nel teardown del test?